jams/core.py

```diff
@@ -1168,6 +1202,34 @@ def slice(self, start_time, end_time, strict=False):
 
         return sliced_ann
 
+    def to_interval_values(self):
+        '''Extract observation data in a `mir_eval`-friendly format.
+
+        Returns
+        -------
+        intervals : np.ndarray [shape=(n, 2), dtype=float]
+            Start- and end-times of all valued intervals
+
+            `intervals[i, :] = [time[i], time[i] + duration[i]]`
+
+        labels : list
+            List view of value field.
+        '''
+
+        times = timedelta_to_float(self.data.time.values)
+        duration = timedelta_to_float(self.data.duration.values)
+
+        return np.vstack([times, times + duration]).T, list(self.data.value)
+
+    def __iter_obs__(self):
+        for _, (t, d, v, c) in self.data.iterrows():
```

Comment:
I find it odd that iterating `self` is (or will be) a shortcut to `self.data.obs` ... which feels like two unnecessary levels of abstraction? right?